### DISCUSSION TITLE
Help Importing Project Files

### DISCUSSION THREAD

**Discussion by MagmaRen** [keiwando/toonsquid#687] [General]
So recently I bought a new Ipad Pro and am trying to import all my project files that I backed up onto google drive but they will not import to ToonSquid on the new IPad. I downloaded them from drive and added them to my files folder on my Ipad but when I go to import them into Toonsquid it doesn’t recognize them. Anyone have any suggestions on what I should do to fix this? Thanks!

**Comment by keiwando** [comment]
Hi!
Could you please share more details about what exactly happens when you try to import the project files into ToonSquid? Are they greyed out in the selection? Are they project files with the .tsproj extension?

Feel free to send a [short screen recording](https://support.apple.com/en-us/102653) that shows the issue to contact@toonsquid.com and I can take a closer look at what's going on. Thanks!

**Comment by MagmaRen** [comment]
Hi, thank you for responding. It turns out some of the files were not saved
as .tsproj files and so they weren’t opening. I have an app that allows me
to modify the file extension so I was able to convert them pack to
Toonsquid project files so now they open in the app. Thank you for reaching
out to assist me. I greatly appreciate it.
